static/js/settings.js

```diff
@@ -672,5 +672,15 @@ exports.setup_page = function () {
     });
 };
 
+exports.star_unstar_user = function (email) {
+    return channel.post({
+        url: "/json/users/me/star_user",
+        data: {"mark_star_email": JSON.stringify(email)},
+        success: function () {
+            activity.update_users();
+        }
+    });
+};
+
```

Comment:
Let's try to find a better place for this function. I think it should go with thinks like muting streams or starring messages.